src/box/flightrec.h

```diff
@@ -21,23 +21,42 @@ extern "C" {
 #include <stddef.h>
 #include <stdint.h>
 
+/** Flight recorder config options. */
+struct flight_recorder_cfg {
+	/**
+	 * Directory to store flight_records.ttfr file.
+	 * Should be the same as memtx_dir.
```

Comment:
Nit: It may be, but it not necessarily should be. Moreover, there's a [ticket](https://github.com/tarantool/tarantool-ee/issues/230) to fix this. Please remove this sentence.